sdk/attestation/attestation/src/models/attestationToken.ts

```diff
@@ -34,27 +46,184 @@ export class AttestationToken
+     *
+     */
+    public get contentType(): string | undefined {
+        return this._header.cty;
+    }
+
+    /**
+     * Json Web Token Header "key URL".
+     *  See [RFC 7515 Section 4.1.2](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.2) for details.
+     *
+     */
+    public get keyUrl() : string | undefined {
+        return this._header.jku;
+    }
+
+    /**
+     * Json Web Token Header "X509 Url".
+     *  See [RFC 7515 Section 4.1.5](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.5) for details.
+     *
+     */
+     public get x509Url() : string | undefined {
+      return this._header.x5u;
+    }
+
+    /**
+     * Json Web Token Header "Typ".
+     *  See [RFC 7515 Section 4.1.9](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.9) for details.
+     *
+     */
+    public get type() : string | undefined {
+      return this._header.typ;
+    }
+    
+    /** Json Web Token Header "x509 thumprint".
+     *  See [RFC 7515 Section 4.1.7](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.7) for details.
+     *
+     */
+    public get certificateThumbprint() : string | undefined {
+      return this._header.x5t;
+    }
+
+    /** Json Web Token Header "x509 SHA256 thumprint".
+     *  See [RFC 7515 Section 4.1.8](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.8) for details.
+     *
+     */
+    public get certificateSha256Thumbprint() : string | undefined {
+      return this._header['x5t#256'];
+    }
+
+    /** Json Web Token Header "x509 certificate chain".
+     *  See [RFC 7515 Section 4.1.6](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.6) for details.
+     *
+     */
+//    public get certificateChain() : AttestationSigner | undefined {
+//        let jwk = new JsonWebKey({x5c: this._header.x5c, kid: this._header.kid});
+//        return new AttestationSigner(jwk);
+//    }
+
+    /*********** JSON WEB TOKEN (RFC 7519) PROPERTIES */
+    
+    
+    /** Issuance time for the token, from JWT body.
+     *   See [RFC 7519 Section 4.1.6](https://www.rfc-editor.org/doc/html/rfc7519#section-4.1.6)
+     *   for details.
+     */
+    public get issuer() : string | undefined {
+      return this._body.iss;
+    }
+
+    /** Expiration time for the token, from JWT body.
+     *   See [RFC 7519 Section 4.1.4](https://www.rfc-editor.org/doc/html/rfc7519#section-4.1.4)
+     *   for details.
+     */
+    public get expirationTime() : Date | undefined {
+        if (this._body.exp) {
+            return new Date(this._body.exp*1000);
+        }
+        return undefined;
+    }
+
+    /** Issuance time for the token, from JWT body.
+     *   See [RFC 7519 Section 4.1.6](https://www.rfc-editor.org/doc/html/rfc7519#section-4.1.6)
+     *   for details.
+     */
+    public get issuedAtTime() : Date | undefined {
+      if (this._body.iat) {
+        return new Date(this._body.iat*1000);
+      }
+      return undefined;
```

Comment:
NIT: this can be a one liner:
```suggestion
        return this._body.iat ? new Date(this._body.iat*1000) : undefined;
```